### ISSUE TITLE
Uploads 2MB limit seems to be very low

### ISSUE THREAD

**Issue by DavidGOrtega** [iterative/cml#232]
https://discord.com/channels/485586884165107732/728693131557732403/751001285100306502

Users might need to upload videos of networks outputs.

@shcheklein

**Comment by shcheklein** [comment]
@DavidGOrtega to my mind we have to put some limit on top of our storage at some point.

@raimannma could you clarify your use case a bit? do you put those video as part of the report? Does GitHub markdown support embedding them?

**Comment by shcheklein** [comment]
@raimannma got it! thanks for sharing the information. For GitLab we definitely should try to use their artifacts storage first. Not sure about GitHub- @DavidGOrtega where does GitHub stands in this? (probably we can do GitLab separately if needed, but would be great to utilize something on the GitHub side).

**Comment by DavidGOrtega** [comment]
@raimannma we have a PR almost ready using uploads api in Gitlab #253

**Comment by DavidGOrtega** [comment]
@raimannma ready! As the command help states the use its the same but adding ```--gitlab-uploads``` parameter.

```sh
cml-publish something.mov --md --gitlab-uploads --title 'this is awesome' >> report.md
```

closed via #233

**Comment by raimannma** [comment]
Thanks, this works perfect for me.
